c7n/mu.py

```diff
@@ -226,9 +226,12 @@ def checksum(fh, hasher, blocksize=65536):
     return hasher.digest()
 
 
-def custodian_archive():
+def custodian_archive(*args):
```

Comment:
can we make this explicit list parameter, wild carding here prevents other parameter usage in future. ie addons=() or packages=()